index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Intento de YouTube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="topNav">
+        <img id="logo" src="./imagenes/logo.jpg" alt="">
+        <div>
+        <input id="search" placeholder="search" type="text">
+        <button>search</button>
+        </div>
+        <div id="profile">
+        <button>notifications</button>
+        <button id="profile">profile</button>
+        </div>
+        
+    </nav>
+    <div id="main">
+        <div id="left">
+    <video controls> 
+        <source src="./videos/ExamenFinal.mp4" type="video/mp4">
+    </video>
+    <div id="title">
+        <h1>Examen?</h1>
+        <div id="like">
+        <button>like</button>
+        <button>don't like</button>
+        <button>share</button>
+        </div>
+    </div>
+    <h3>21035 views      3/03/2022</h3>
+    <h3>Siempre que pienso en la universidad</h3>
```

Comment:
This sounds rude to me. Please be polite in your comments. You should give suggestions and explain why they are given. Here you just gave an explanation, which is by the way not very explanatory. Example:
> Kindly replace this text with an English sentence because it is require by the project requirements.

Reply:
Kindly replace this text with an English sentence because it is require by the project requirements.